src/funfuzz/js/compile_shell.py

```diff
@@ -250,8 +248,10 @@ def ensureDir(directory):  # pylint: disable=invalid-name,missing-param-doc,miss
 def autoconfRun(cwDir):  # pylint: disable=invalid-name,missing-param-doc,missing-type-doc
     """Run autoconf binaries corresponding to the platform."""
     if sps.isMac:
-        autoconf213_mac_bin = '/usr/local/Cellar/autoconf213/2.13/bin/autoconf213' \
-                              if sps.isProgramInstalled('brew') else 'autoconf213'
+        if subprocess.check_call(["which", "brew"], stdout=subprocess.PIPE):
```

Comment:
Why set a static path to autoconf after calling _which_?  Shouldn't you be parsing out the path from _which_?

Reply:
homebrew does not add itself to the system PATH, meaning nothing happens when you run `autoconf213` and thus nothing shows when we run `which autoconf213`. Other package managers do, though, e.g. MacPorts, but homebrew is the recommended way to do things now.